internal/configs/version2/nginx-plus.virtualserver.tmpl

```diff
@@ -1,3 +1,8 @@
+map $http_upgrade $connection_header {
```

Comment:
this map is defined in the http context, which is shared among the generated config for all Ingress and VS resources. For example, if you have multiple VS resources, the generated config for each resource will include this map. As a result, you will get a conflict and NGINX will fail to reload.

You need to move this map into the main NGINX configuration file.